cli/pack/common.go

```diff
@@ -534,3 +534,29 @@ func copyBinaries(srcPath, destPath string) error {
 
 	return nil
 }
+
+// getPackageName returns the result name of the package.
+func getPackageName(packCtx *cmdcontext.PackCtx, suffix string, addVersion bool) (string, error) {
+	var tarName, versionSuffix string
+	if addVersion {
+		versionSuffix = getVersion(packCtx)
+	}
+
+	if packCtx.FileName != "" {
+		return packCtx.FileName, nil
```

Comment:
Looks like, this must be checked first. Before `if addVersion`

Reply:
Right, fixed.